webpack.config.js

```diff
@@ -32,13 +31,16 @@ function commonConfig() {
 
 var configs = [];
 var progConfig = commonConfig();
-progConfig.entry = path.join(__dirname, './impexp/createUnitsFromXmlEssence.js'),
-progConfig.output.filename = './impexp/createUnitsFromXmlEssenceBuild.js'
-configs.push(progConfig);
-
-progConfig = commonConfig();
-progConfig.entry = path.join(__dirname, './impexp/updateData.js'),
-progConfig.output.filename = './impexp/updateDataBuild.js'
+progConfig.entry = path.join(__dirname, './impexp/updateData.js');
+progConfig.externals = [
+    function (context, request, callback) {
+      if(/ucumDefs\.json/.test(request)) {
+        return callback(null, 'commonjs ' + request);
```

Comment:
I think this will result in loading the existing ucumDefs.jon file (prior to updating it).  I am not sure what effects that will have (maybe none), but I suspect that this externals declaration does not make a difference in the output ucumDefs.json.

Reply:
This excludes ucumDefs.json from updateDataBuild.js bundle for "action 2". And force to load ucumDefs.json from data folder every time when you run "action 2" from updateDataBuild.

Reply:
Sorry, now I understand what you mean - I will fix it soon.